configuration.k

```diff
@@ -135,10 +135,10 @@ imports SOLIDITY-SYNTAX
                         <ctLocation> .Map </ctLocation>
                         <tempContext> .Map </tempContext>
                         <tempStore> .Map </tempStore>
-                        <tempdistore> .Map </tempdistore>
-                        <tempmemory> .Map </tempmemory>
-                        <ditype> .Map </ditype>
-                        <distore> .Map </distore>
+                        <tempIdstore> .Map </tempIdstore>
```

Comment:
tempIdStore?

Reply:
Temporary dynamic store in the contract instance.

Reply:
same as above, I just meant that it should use capital letters to separate "temp", "id" and "store", as in `tempIdStore` (as opposed to `tempIdstore`). Nothing major, but it's worth keeping the same convention everywhere :)